cookbook/templates/default/baragon-agent.init.erb

```diff
@@ -0,0 +1,10 @@
+description "Baragon Agent Service"
+author "EverTrue"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+respawn limit 10 5
+
+exec <%= node['java']['java_home'] %>/bin/java -Xmx512m -Djava.net.preferIPv4Stack=true -jar /usr/share/java/<%= @baragon_jar %> server <%= @config_yaml %> >> /var/log/baragon/baragon_agent.log 2>&1
```

Comment:
I'm going to leave the heap size static since that's only likely to change if someone who has commit access to this repo changes something, but the log file should probably be an attribute.